components/dash-core-components/tests/integration/input/test_input_basics.py

```diff
@@ -72,3 +73,34 @@ def test_inbs002_user_class(dash_dcc):
     dash_dcc.wait_for_style_to_equal(".test-input-css input", "width", "420px")
 
     assert dash_dcc.get_logs() == []
+
+
+def test_inbs003_styles_are_scoped(dash_dcc):
+    app = Dash(__name__)
+
+    app.index_string = """
+    <html>
+        <body>
+            <input id="ExternalInput" required />
+            {%app_entry%}
+            {%config%}
+            {%scripts%}
+            {%renderer%}
+        </body>
+    </html>
+    """
+
+    app.layout = html.Div(
+        className="test-input-css",
+        children=[dcc.Input(id="DashInput", required=True, className="unittest")],
+    )
+
+    dash_dcc.start_server(app)
+
+    external_input = dash_dcc.driver.find_element(By.ID, "ExternalInput")
```

Comment:
You can use `dash_dcc.find_element("#ExternalInput")` instead of the driver and By object.

Reply:
Addressed in 745f0f3